### PR TITLE
Recalibrate the temperature sensor every 5 minutes

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

**Comment by nomis52** [comment]
Closes #230

File: firmware/src/temperature.c [hunk 1]
```diff
@@ -28,6 +28,8 @@
 #include "app_settings.h"
 
 static const uint16_t SAMPLING_PERIOD = 10000;  // 1s
+// Recalibrate every 5 minutes
```
Comment (peternewman) [inline firmware/src/temperature.c]: Can you expand the comment to clarify it's 300 * SAMPLING_PERIOD, which happens to be in seconds.

**Comment by peternewman** [comment]
Generally LGTM

**Comment by peternewman** [comment]
LGTM

**Pull request merged**